source/resiliency.txt

```diff
@@ -12,12 +12,140 @@ Application and Database Resiliency
    :depth: 1
    :class: onecol
 
-Intro statement
+MongoDB |service| is a modern, highly-performant database that is purpose-built 
+to maintain uptime regardless of infrastructure outages, system maintenance and more.
 
 {+service+} Features and Best Practices for Resiliency
 ------------------------------------------------------
 
-Content here
+Database Replication
+~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Changing this for parity with other pages. Can you adjust the rest of the heading levels below this as well please?
```suggestion
Features
~~~~~~~

Database Replication
````````````````````

```